CONTRIBUTING.md

```diff
@@ -38,6 +38,15 @@ pip install -e '.[dev]'
 pytest
 ```
 
+If your changes could affect solver (or compilation) performance, please run the
+benchmark tests with 
```

Comment:
+mention that they could run benchmarks before/after their change to test it.

Honestly I am starting to warm to the idea of using a service like codspeed etc to log benchmark results over time.